The CI system is telling me that your files are not valid JSON:

```
FAIL: test_all_files_JSON_content (tests.test_all_json.TestAllFilesAreJSON)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/data/tests/test_all_json.py", line 31, in test_all_files_JSON_content
    self.assertEqual(not_json, [])
AssertionError: Lists differ: ['75-guessable-credentials.jso... != []

First list contains 2 additional elements.
First extra element 0:
75-guessable-credentials.json

+ []
- ['75-guessable-credentials.json',
-  '76-x-content-type-options_header_missing.json']
```

https://circleci.com/gh/vulndb/data/51

--- a/db/75-guessable-credentials.json
+++ b/db/75-guessable-credentials.json
@@ -4,7 +4,7 @@
   "severity": "low", 
   "description": [
     "Web Applications often have predictable usernames and weak password policies set.", 
-    " This can easily allow an attacker to gain unauthorised access into the application", 
+    " This can easily allow an attacker to gain unauthorized access into the application", 
     "by guessing the credentials used for authentication.\n\n It was possible to guess and",
     "gain a set a valid credentials for the application is scope."
   ], 

--- a/db/75-guessable-credentials.json
+++ b/db/75-guessable-credentials.json
@@ -14,7 +14,7 @@
       "Do not have any default credentials set on the application. Any known usernames", 
       "or passwords associated with the application framework should also be removed.", 
       "Furthermore, Web Applications should implement a strong password policy consisting", 
-      "of a combination of alphanumeric characters and a minimum length of 8 characters.", 
+      "of a combination of alphanumeric characters and a minimum length of 8 characters." 
     ]
   },
   "cwe": ["522"],

--- a/db/76-x-content-type-options_header_missing.json
+++ b/db/76-x-content-type-options_header_missing.json
@@ -7,7 +7,7 @@
     "content-sniffing attacks in Internet Explorer and Google Chrome. MIME content-sniffing", 
     "is a mechanism that allows browsers to inspect and dynamically guess the content",
     "type and file type.\n\nMIME Sniffing checking algorithm has known problems which",
-    "might allow users to upload files that might contain malicous code.If an attacker"
+    "might allow users to upload files that might contain malicous code. If an attacker"
     "can spoof a file type/content and upload it to the application successfully, it is"
     "possible to inject malicous code which can be downloaded and viewed by other "
     "users of the application. This can lead to attacks such as persistant Cross-Site Scripting."

